libs/core/langchain_core/language_models/base.py

```diff
@@ -59,9 +60,14 @@ def _get_token_ids_default_method(text: str) -> List[int]:
 LanguageModelLike = Runnable[LanguageModelInput, LanguageModelOutput]
 LanguageModelOutputVar = TypeVar("LanguageModelOutputVar", BaseMessage, str)
 
+_OutputSchema = TypeVar("_OutputSchema")
+_OutputFormat = TypeVar("_OutputFormat")
```

Comment:
should we restrict these a bit? Right now could be anything right? Seems difficult to use with both.

Just an output base model as well and force it to be pydantic or general json type (list/dict/etc) doesn't seem too bad to me